### ISSUE TITLE
SlowCheetah and SlowCheetah.VS localization pass

### ISSUE THREAD

**Issue by jviau** [Microsoft/slow-cheetah#19]
[SlowCheetah](../tree/master/src/Microsoft.VisualStudio.SlowCheetah/MultilingualResources)
[SlowCheetah.VS](../tree/master/src/Microsoft.VisualStudio.SlowCheetah.VS/MultilingualResources)

For now, localization will be done in the [localization branch](https://github.com/Microsoft/slow-cheetah/tree/localization).

**Comment by janbaltus** [comment]
Localized files under
https://github.com/Microsoft/slow-cheetah/pull/28

**Comment by davilimap** [comment]
@janbaltus After some large changes to the project structure, I created a [new localization branch](https://github.com/Microsoft/slow-cheetah/tree/loc) and brought over the changes from #28 to avoid issues with the newly generated files. There are some new strings that require localization, is it possible to to do another pass on the new branch? I've also specified the resources that don't need translation.
